hub/util/timer.py

```diff
@@ -0,0 +1,12 @@
+import time
+
+
+class Timer:
```

Comment:
Let's please not re-introduce this back into 2.0. I hate this class. We can get away with using pytest-benchmark in most, if not all, cases. Where is this _necessary_?

Reply:
it's not necessary but I find it convenient, compared to pytest-benchmark, for quickly profiling different parts of a single function. Not worth a debate though, removing it for now